packages/overlay/src/App.tsx

```diff
@@ -59,10 +59,18 @@ export default function App({
       setOpen(false);
     };
 
+    const handleKeyPress = (event: KeyboardEvent) => {
+      if (event && event.ctrlKey && event.shiftKey && event.key.toLowerCase() === 's') {
```

Comment:
Let's accept the Cmd key on MacOS as well. CTRL on macbooks is a bit unintuitive. Seems like this is safe to use because it [doesn't map to the windows key
](https://stackoverflow.com/a/3922353). I'm fine with both combinations working on MacOS.

```suggestion
      if (event && (event.ctrlKey || event.metaKey) && event.shiftKey && event.key.toLowerCase() === 's') {
```